asab/storage/service.py

```diff
@@ -27,6 +27,16 @@ def upsertor(self, collection:str, obj_id, version=None):
 		pass
 
 
+	@abc.abstractmethod
+	async def list(self, collection:str):
+		pass
```

Comment:
Document.

Reply:
`async def list(self, collection:str) -> collection.abc.Itertor:`

... I strongly believe.

Reply:
`mongodb` strorage return `motor.motor_asyncio.AsyncIOMotorCursor` while `inmemory` returns list. I added docs to `asab.storage.mongodb/inmemory.StorageService.list`

Reply:
If both returned objects comply to `collections.abc.Iterator` then it is fine.

Reply:
`AsyncIOMotorCursor` inherits from `AgnosticCursor` which inherits from `AgnosticBaseCursor` which inherits from `AgnosticBase` which inherits from object.